identity/app/Global.scala

```diff
@@ -1,40 +0,0 @@
-import common.{CloudWatchMetricsLifecycle, LifecycleComponent, BackwardCompatibleLifecycleComponents}
-import common.Logback.LogstashLifecycle
-import conf._
-import conf.switches.SwitchboardLifecycle
-import controllers.HealthCheck
-import model.ApplicationIdentity
-import play.api._
-import play.api.inject.ApplicationLifecycle
-import play.api.inject.guice._
-
-import scala.concurrent.ExecutionContext
-
-object Global extends GlobalSettings with BackwardCompatibleLifecycleComponents {
-
-  override def lifecycleComponents(appLifecycle: ApplicationLifecycle)(implicit ec: ExecutionContext): List[LifecycleComponent] = List(
-    new CloudWatchMetricsLifecycle(appLifecycle, ApplicationIdentity("frontend-identity")),
-    new IdentityLifecycle(appLifecycle),
-    new SwitchboardLifecycle(appLifecycle),
-    LogstashLifecycle,
-    new CachedHealthCheckLifeCycle(HealthCheck)
-  )
-}
-
-class IdentityApplicationLoader extends GuiceApplicationLoader() {
-
-  override def builder(context: ApplicationLoader.Context): GuiceApplicationBuilder = {
-    val module = context.environment.mode match {
-      case Mode.Prod => {
```

Comment:
This is also handled here https://github.com/guardian/frontend/pull/13571/commits/b567fb13a556e444b8b6ed1719dfb94aca2cf32d#diff-9eb1db9165a23dea3adc8ceceba582c5R12